src/Sarif.Converters/CppCheckError.cs

```diff
@@ -111,11 +106,6 @@ public static CppCheckError Parse(XmlReader reader, CppCheckStrings strings)
         /// <returns>This instance as an <see cref="Result"/>.</returns>
         public Result ToSarifIssue()
         {
-            if (this.Locations.Length == 0)
-            {
-                throw new InvalidOperationException("At least one location must be present in a SARIF result.");
```

Comment:
>location [](start = 66, length = 8)

This check was definitely wrong. It is absolutely not required for a SARIF `result` to have a `locations` property.